sycl/include/sycl/detail/pi.h

```diff
@@ -630,6 +635,8 @@ static constexpr pi_device_partition_property
     PI_DEVICE_PARTITION_BY_COUNTS_LIST_END = 0x0;
 static constexpr pi_device_partition_property
     PI_DEVICE_PARTITION_BY_AFFINITY_DOMAIN = 0x1088;
+static constexpr pi_device_partition_property
+    PI_DEVICE_EXT_INTEL_PARTITION_BY_CSLICE = 0x1089;
```

Comment:
```suggestion
    PI_EXT_INTEL_DEVICE_PARTITION_BY_CSLICE = 0x1089;
```

Reply:
Why do you not add any handling to plugins other than L0?

Reply:
Are you asking about reporting this scheme as unsupported or actually implementing the support? I'd expect it's about the former and for that I assumed that any reasonable implementation would return proper error code for unsupported partitioning scheme already, similarly to how the level zero plugin behaves (both before/after this change).

Reply:
> I assumed that any reasonable implementation would return proper error code for unsupported partitioning scheme already

Please double check this.

Reply:
cuda/esimd/hip don't support any kind of partitioning, so no changes are needed.

OpenCL directly delegates to `clCreateSubDevices` and it would return "a NULL value with the following error values returned in errcode_ret:"

> CL_​INVALID_​VALUE if values specified in properties are not valid or if values specified in properties are valid but not supported by the device.